MailCore/Utils/SwiftSoupUtils.swift

```diff
@@ -18,17 +18,43 @@
 
 import SwiftSoup
 
-public enum SwiftSoupUtils {
-    public static func extractHTML(from document: Document, _ cssQuery: String) throws -> String {
-        guard let foundElement = try document.select(cssQuery).first() else {
-            throw SwiftSoupError.elementNotFound
+public struct SwiftSoupUtils {
```

Comment:
As we clean a notification, here we are fine, but what do you think of marking `functions` of this class async ?

`extractText` is already async, but it looks like this is the only one.

That way, if work is triggered from a view, it will not be run on the main queue.